libs/bluegl/CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ add_executable(test_${TARGET}
 
 if (LINUX)
     target_link_libraries(test_${TARGET} PUBLIC dl)
+    if (EGL)
```

Comment:
There's no need to link against EGL (see below)

Reply:
removed